integrations/huggingface-transformers/tutorials/bert_sparse_transfer_learning.md

```diff
@@ -0,0 +1,149 @@
+
+The use cases and datasets covered in this tutorial are listed below along with results for sparse transfer learning with the 80% pruned BERT model.
+
+| Use Case                   | Dataset                                                                       | Description                                                                                                                                                                                                          | Sparse Transfer Results  |
+|----------------------------|-------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|
+| Question and Answering     | [SQuAD](https://rajpurkar.github.io/SQuAD-explorer/)                          | A reading comprehension dataset, consisting of questions posed by crowdworkers on a set of Wikipedia articles, where the answer to every question is a segment of text from the corresponding reading passage.       | 87.1 F1 (87.5 baseline)  |
+| Binary Classification      | [QQP](https://quoradata.quora.com/First-Quora-Dataset-Release-Question-Pairs) | A dataset made up of potential question pairs from Quora with a boolean label representing whether or not the questions are duplicates                                                                              | 90 acc (91.3 baseline)   |
+| Multi-Class Classification | [MNLI](https://cims.nyu.edu/~sbowman/multinli/)                               | A crowd-sourced collection of sentence pairs annotated with textual entailment information. It covers a range of genres of spoken and written text and supports a distinctive cross-genre generalization evaluation. | 81.9 acc (82.3 baseline) |
+| Named Entity Recognition   | [CoNNL2003](https://www.clips.uantwerpen.be/conll2003/ner/)                   | A dataset concentrated on four types of named entities: persons, locations, organizations, and names of miscellaneous entities that do not belong to the previous three groups.                                       | 88.5 acc (94.7 baseline) |
+| Sentiment Analysis         | [SST2](https://nlp.stanford.edu/sentiment/)                                   | A corpus that includes fine-grained sentiment labels for phrases within sentences and presents new challenges for sentiment compositionality.                                                                       | 91.3 acc (91.2 baseline) |
+
+Once you have selected a use case and dataset, you are ready to create a teacher model.
+
+## Creating a Dense Teacher
+
+Distillation works very well for BERT and NLP in general to create highly sparse and accurate models for deployment.
+Following this sentiment, you will create a dense teacher model before applying sparse transfer learning.
```

Comment:
In practice it's highly likely that the client already has a dense model trained on the (downstream) dataset and wish to sparsify. If so it could be used as the teacher. We should mention this here.

Reply:
Good point, will update